#353 Breaks tests on Mac OS X

After #353 merged I can't run tests locally on my mac.

logs:
```
--> Installing build dependencies
--> Building nomad
--> Running tests
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17002 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17002.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17002 updated peer set (2): [127.0.0.1:17002]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient426611545
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17002.global (Addr: 127.0.0.1:17002) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AgentSelf (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17005.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17005 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [system service batch _core]
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient587818739
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17005 updated peer set (2): [127.0.0.1:17005]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17005.global (Addr: 127.0.0.1:17005) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AgentJoin (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17008 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17008.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17008 updated peer set (2): [127.0.0.1:17008]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient701296029
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17008.global (Addr: 127.0.0.1:17008) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AgentMembers (0.03s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17011.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17011 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17011 updated peer set (2): [127.0.0.1:17011]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [batch system service _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17011.global (Addr: 127.0.0.1:17011) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient991152087
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AgentForceLeave (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17014.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17014 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17014 updated peer set (2): [127.0.0.1:17014]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient901646881
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17014.global (Addr: 127.0.0.1:17014) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AgentSetServers (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17017.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17017 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17017 updated peer set (2): [127.0.0.1:17017]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17017.global (Addr: 127.0.0.1:17017) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient103179515
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestAgent_RPCPing (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17020 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17020.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17020 updated peer set (2): [127.0.0.1:17020]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [system service batch _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17020.global (Addr: 127.0.0.1:17020) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient144067301
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AllocsList (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17023.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17023 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17023 updated peer set (2): [127.0.0.1:17023]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient340869215
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17023.global (Addr: 127.0.0.1:17023) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_AllocQuery (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
handleReap
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17026 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17026.global 127.0.0.1
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17026 updated peer set (2): [127.0.0.1:17026]
handleReap
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [batch system service _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17026.global (Addr: 127.0.0.1:17026) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient740212904
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_EvalList (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17029 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17029.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17029 updated peer set (2): [127.0.0.1:17029]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient979641370
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17029.global (Addr: 127.0.0.1:17029) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_EvalAllocations (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17032 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17032.global 127.0.0.1
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17032 updated peer set (2): [127.0.0.1:17032]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [system service batch _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient604586076
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17032.global (Addr: 127.0.0.1:17032) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_EvalQuery (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17035 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17035 updated peer set (2): [127.0.0.1:17035]
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17035.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [batch system service _core]
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17035.global (Addr: 127.0.0.1:17035) (DC: dc1)
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient866365934
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestContentTypeIsJSON (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17038 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17038.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17038 updated peer set (2): [127.0.0.1:17038]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [system service batch _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient256707920
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17038.global (Addr: 127.0.0.1:17038) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestPrettyPrint (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17041.global 127.0.0.1
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17041 [Leader] entering Leader state
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17041 updated peer set (2): [127.0.0.1:17041]
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient532921602
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17041.global (Addr: 127.0.0.1:17041) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestPrettyPrintBare (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17044 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17044.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17044 updated peer set (2): [127.0.0.1:17044]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17044.global (Addr: 127.0.0.1:17044) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient604316036
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestParseRegion (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17047 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17047.global 127.0.0.1
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17047 updated peer set (2): [127.0.0.1:17047]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient798139222
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17047.global (Addr: 127.0.0.1:17047) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobsList (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17050.global 127.0.0.1
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17050 [Leader] entering Leader state
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17050 updated peer set (2): [127.0.0.1:17050]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient006831864
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17050.global (Addr: 127.0.0.1:17050) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobsRegister (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17053 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17053.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17053 updated peer set (2): [127.0.0.1:17053]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17053.global (Addr: 127.0.0.1:17053) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient285399786
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobQuery (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17056 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17056 updated peer set (2): [127.0.0.1:17056]
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17056.global 127.0.0.1
handleReap
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient913570732
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17056.global (Addr: 127.0.0.1:17056) (DC: dc1)
2015/10/29 12:52:28 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobUpdate (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:28 [INFO] raft: Node at 127.0.0.1:17059 [Leader] entering Leader state
2015/10/29 12:52:28 [INFO] serf: EventMemberJoin: Node 17059.global 127.0.0.1
2015/10/29 12:52:28 [INFO] raft: Disabling EnableSingleNode (bootstrap)
handleReap
2015/10/29 12:52:28 [DEBUG] raft: Node 127.0.0.1:17059 updated peer set (2): [127.0.0.1:17059]
2015/10/29 12:52:28 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:28 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:28 [INFO] nomad: adding server Node 17059.global (Addr: 127.0.0.1:17059) (DC: dc1)
2015/10/29 12:52:28 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient478471102
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobDelete (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17062 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17062 updated peer set (2): [127.0.0.1:17062]
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17062.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient335642016
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17062.global (Addr: 127.0.0.1:17062) (DC: dc1)
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobForceEvaluate (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17065 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17065.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [batch system service _core]
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17065 updated peer set (2): [127.0.0.1:17065]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17065.global (Addr: 127.0.0.1:17065) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient765540818
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobEvaluations (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17068.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17068 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17068 updated peer set (2): [127.0.0.1:17068]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17068.global (Addr: 127.0.0.1:17068) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient489820372
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_JobAllocations (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17071 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17071.global 127.0.0.1
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
handleReap
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17071 updated peer set (2): [127.0.0.1:17071]
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17071.global (Addr: 127.0.0.1:17071) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient871761190
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_NodesList (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17074 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17074.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17074 updated peer set (2): [127.0.0.1:17074]
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [system service batch _core]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17074.global (Addr: 127.0.0.1:17074) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient997522760
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_NodeForceEval (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17077 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17077.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17077 updated peer set (2): [127.0.0.1:17077]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient319787962
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17077.global (Addr: 127.0.0.1:17077) (DC: dc1)
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_NodeAllocations (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17080.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17080 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17080 updated peer set (2): [127.0.0.1:17080]
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17080.global (Addr: 127.0.0.1:17080) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient189426940
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_NodeDrain (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17083 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17083.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17083 updated peer set (2): [127.0.0.1:17083]
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient631537550
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17083.global (Addr: 127.0.0.1:17083) (DC: dc1)
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_NodeQuery (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17086.global 127.0.0.1
handleReap
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17086 [Leader] entering Leader state
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17086 updated peer set (2): [127.0.0.1:17086]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient996022768
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17086.global (Addr: 127.0.0.1:17086) (DC: dc1)
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_StatusLeader (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
2015/10/29 12:52:29 [INFO] serf: EventMemberJoin: Node 17089.global 127.0.0.1
2015/10/29 12:52:29 [INFO] raft: Node at 127.0.0.1:17089 [Leader] entering Leader state
handleReap
2015/10/29 12:52:29 [INFO] raft: Disabling EnableSingleNode (bootstrap)
2015/10/29 12:52:29 [INFO] nomad: starting 8 scheduling worker(s) for [service batch system _core]
2015/10/29 12:52:29 [DEBUG] raft: Node 127.0.0.1:17089 updated peer set (2): [127.0.0.1:17089]
2015/10/29 12:52:29 [INFO] nomad: cluster leadership acquired
2015/10/29 12:52:29 [INFO] nomad: adding server Node 17089.global (Addr: 127.0.0.1:17089) (DC: dc1)
2015/10/29 12:52:29 [INFO] client: using alloc directory /var/folders/lw/kb0_gx6x43d6jv3mj4s4t6v00000gn/T/NomadClient252441762
2015/10/29 12:52:29 [WARN] fingerprint.network: Error calling ifconfig (/sbin/ifconfig lo): exit status 1
--- FAIL: TestHTTP_StatusPeers (0.04s)
	agent_test.go:67: err: client setup failed: fingerprinting failed: Unable to find any network interface which has IP address
FAIL
coverage: 49.5% of statements
FAIL	github.com/hashicorp/nomad/command/agent	1.174s
make: *** [test] Error 1
```

my ifconfig output:
```
lo0: flags=8049<UP,LOOPBACK,RUNNING,MULTICAST> mtu 16384
	options=3<RXCSUM,TXCSUM>
	inet6 ::1 prefixlen 128
	inet 127.0.0.1 netmask 0xff000000
	inet6 fe80::1%lo0 prefixlen 64 scopeid 0x1
	nd6 options=1<PERFORMNUD>
gif0: flags=8010<POINTOPOINT,MULTICAST> mtu 1280
stf0: flags=0<> mtu 1280
en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 80:e6:50:0a:78:44
	inet6 fe80::82e6:50ff:fe0a:7844%en0 prefixlen 64 scopeid 0x4
	inet 172.31.15.123 netmask 0xfffff800 broadcast 172.31.15.255
	nd6 options=1<PERFORMNUD>
	media: autoselect
	status: active
en1: flags=963<UP,BROADCAST,SMART,RUNNING,PROMISC,SIMPLEX> mtu 1500
	options=60<TSO4,TSO6>
	ether 72:00:04:b3:31:40
	media: autoselect <full-duplex>
	status: inactive
en2: flags=963<UP,BROADCAST,SMART,RUNNING,PROMISC,SIMPLEX> mtu 1500
	options=60<TSO4,TSO6>
	ether 72:00:04:b3:31:41
	media: autoselect <full-duplex>
	status: inactive
p2p0: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 2304
	ether 02:e6:50:0a:78:44
	media: autoselect
	status: inactive
awdl0: flags=8943<UP,BROADCAST,RUNNING,PROMISC,SIMPLEX,MULTICAST> mtu 1484
	ether da:39:5f:96:30:d3
	inet6 fe80::d839:5fff:fe96:30d3%awdl0 prefixlen 64 scopeid 0x8
	nd6 options=1<PERFORMNUD>
	media: autoselect
	status: active
bridge0: flags=8822<BROADCAST,SMART,SIMPLEX,MULTICAST> mtu 1500
	options=63<RXCSUM,TXCSUM,TSO4,TSO6>
	ether 82:e6:50:a0:c3:00
	Configuration:
		id 0:0:0:0:0:0 priority 0 hellotime 0 fwddelay 0
		maxage 0 holdcnt 0 proto stp maxaddr 100 timeout 1200
		root id 0:0:0:0:0:0 priority 0 ifcost 0 port 0
		ipfilter disabled flags 0x2
	member: en1 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 5 priority 0 path cost 0
	member: en2 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 6 priority 0 path cost 0
	media: <unknown type>
	status: inactive
vboxnet0: flags=8842<BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 0a:00:27:00:00:00
vboxnet1: flags=8842<BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 0a:00:27:00:00:01
vboxnet2: flags=8842<BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 0a:00:27:00:00:02
vboxnet3: flags=8842<BROADCAST,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 0a:00:27:00:00:03
vboxnet4: flags=8943<UP,BROADCAST,RUNNING,PROMISC,SIMPLEX,MULTICAST> mtu 1500
	ether 0a:00:27:00:00:04
	inet 192.168.99.1 netmask 0xffffff00 broadcast 192.168.99.255
fw0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 4078
	lladdr 00:0a:27:02:00:63:7c:20
	nd6 options=1<PERFORMNUD>
	media: autoselect <full-duplex>
	status: inactive
en7: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	options=10b<RXCSUM,TXCSUM,VLAN_HWTAGGING,AV>
	ether ac:87:a3:24:97:a3
	inet6 fe80::ae87:a3ff:fe24:97a3%en7 prefixlen 64 scopeid 0x10
	inet 192.168.36.158 netmask 0xfffffc00 broadcast 192.168.39.255
	nd6 options=1<PERFORMNUD>
	media: autoselect (1000baseT <full-duplex>)
	status: active
```

--- a/command/agent/config.go
+++ b/command/agent/config.go
@@ -7,6 +7,7 @@ import (
 	"net"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
 
 	"github.com/hashicorp/hcl"
@@ -220,7 +221,12 @@ func DevConfig() *Config {
 	conf.DevMode = true
 	conf.EnableDebug = true
 	conf.DisableAnonymousSignature = true
-	conf.Client.NetworkInterface = "lo"
+	if runtime.GOOS == "darwin" {
+		conf.Client.NetworkInterface = "lo0"
+	} else {
+		conf.Client.NetworkInterface = "lo"
+	}
+
 	return conf
 }
 

